core/src/main/java/ar/com/wolox/wolmo/core/activity/WolmoActivityHandler.kt

```diff
@@ -19,49 +19,40 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package ar.com.wolox.wolmo.core.activity;
+package ar.com.wolox.wolmo.core.activity
 
-import android.os.Bundle;
+import android.os.Bundle
+import ar.com.wolox.wolmo.core.R
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import javax.inject.Inject
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import javax.inject.Inject;
-
-import ar.com.wolox.wolmo.core.R;
-import ar.com.wolox.wolmo.core.util.ToastFactory;
-
-public class WolmoActivityHandler {
-
-    private ToastFactory mToastFactory;
-    private WolmoActivity mWolmoActivity;
-
-    @Inject
-    WolmoActivityHandler(ToastFactory toastFactory) {
-        mToastFactory = toastFactory;
-    }
+class WolmoActivityHandler @Inject constructor(
+        private val mToastFactory: ToastFactory
+) {
+    private lateinit var wolmoActivity: WolmoActivity
 
     /**
      * Handles the custom lifecycle of Wolmo's Activity. It provides a set of callbacks to structure
      * the different aspects of the Activities initialization.
      *
-     * @param savedInstanceState a {@link Bundle} provided by Android's lifecycle.
-     * @param activity the {@link WolmoActivity} to be managed by this handler.
+     * savedInstanceState a [Bundle] provided by Android's lifecycle.
```

Comment:
Also remember [Kdoc convention](https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments) says "incorporate the description of parameters and return values directly into the documentation comment".
I write an example about it:
```
/**
* Handles the custom lifecycle of Wolmo's Activity. It provides a set of callbacks to structure
* the different aspects of the Activities initialization. It expects the [activity] to be managed by te
* this and the [savedInstanceState] provided by Android's lifecycle.
*/

Reply:
thanks! done!